app/routes_for_rentals.py

```diff
@@ -0,0 +1,85 @@
+from flask import Blueprint, json, jsonify, request, make_response
+from sqlalchemy.sql.expression import distinct
+from app.models.customer import Customer
+from app.models.video import Video
+from app.models.rental import Rental
+from app import db
+from flask_sqlalchemy import model
+from sqlalchemy import func
+import requests
+from datetime import timedelta, datetime
+
+rentals_bp = Blueprint("rentals", __name__, url_prefix="/rentals")
+
+@rentals_bp.route("/check-out", methods=["POST"])
+def handle_rentals():
+    request_body=request.get_json()
+    if not request_body.get("video_id") or not request_body.get("customer_id"):
+        return make_response({"message":"Could not perform checkout"}, 400)
+
+    video = Video.query.get(request_body["video_id"])
+    customer = Customer.query.get(request_body["customer_id"])
+    if not customer:
+        return make_response({"message":"Could not perform checkout"}, 404)
+    elif not video:
+        return make_response({"message":"Could not perform checkout"}, 404)
```

Comment:
We can combine these conditionals:

```suggestion
    if not customer or not video:
        return make_response({"message":"Could not perform checkout"}, 404)
```